tests/rtl/integration/LoginPageRTL.spec.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { axe } from 'jest-axe';
+import { screen, fireEvent, waitFor } from '@testing-library/react';
```

Comment:
```suggestion
import { screen, fireEvent } from '@testing-library/react';
```
waitFor is unused in this file, but beyond that, I notice that `eslint --fix` that is implement on `lint-staged` is not already consider `webpack` and `tests`, maybe you can add `tests` to existing `lint:code` and then call `yarn lint` in `lint-staged` with `run-script` or the workaround that lint-staged need to run local script.
In this way before each commic you will lint `src`, `webpack` and `tests` folders and  `style-lint` instead of only `src` folder

Reply:
Nice catch!